http-clients/netty-nio-client/src/test/java/software/amazon/awssdk/http/nio/netty/internal/ProxyTunnelInitHandlerTest.java

```diff
@@ -52,6 +35,15 @@
 import org.mockito.Mock;
 import org.mockito.runners.MockitoJUnitRunner;
 
+import java.io.IOException;
+import java.net.URI;
+import java.util.Base64;
+import java.util.function.Supplier;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.*;
```

Comment:
We don't use wildcard imports in general. We do provide a `intellij-codestyle.xml` that helps formatting the code automatically according to our code style guidelines in Intellij. 

Can you apply the codestyle file to your code? The file can be found here, along with other helpful information: https://github.com/aws/aws-sdk-java-v2/blob/master/docs/GettingStarted.md